types.py

```diff
@@ -1,7 +1,10 @@
-from LSP.plugin.core.typing import Any, Callable, List, Literal, TypedDict, TypeVar
+from LSP.plugin.core.typing import Any, Callable, List, Literal, Tuple, TypedDict, TypeVar
 
 T_Callable = TypeVar("T_Callable", bound=Callable[..., Any])
 
+StPoint = int
+StRegionTuple = Tuple[int, int]
```

Comment:
I think technically it should be
```suggestion
StRegionTuple = Tuple[StPoint, StPoint]
```